tests/client.test.ts

```diff
@@ -64,6 +85,75 @@ describe('DBOSClient', () => {
     await DBOS.shutdown();
   });
 
+  test('enqueue-timeout-simple', async () => {
+    const client = await DBOSClient.create(database_url);
+    const wfid = randomUUID();
+
+    await DBOS.launch();
+
+    try {
+      const handle = await client.enqueue<typeof ClientTest.blockingWorkflow>({
+        workflowName: 'blockingWorkflow',
+        workflowClassName: 'ClientTest',
+        queueName: 'testQueue',
+        workflowID: wfid,
+        workflowTimeout: 1000,
+      });
+      await expect(handle.getResult()).rejects.toThrow(new DBOSWorkflowCancelledError(wfid));
+
+      const wfstatus = await client.getWorkflow(wfid);
+      expect(wfstatus?.status).toBe('CANCELLED');
```

Comment:
Check the child is cancelled too.

Reply:
Not for this test (there is no child) but I can check on other tests